sklearn/model_selection/_search.py

```diff
@@ -1200,8 +1210,11 @@ class RandomizedSearchCV(BaseSearchCV):
         FitFailedWarning is raised. This parameter does not affect the refit
         step, which will always raise the error.
 
-    return_train_score : boolean, default=True
-        If ``'False'``, the ``cv_results_`` attribute will not include training
+    return_train_score : boolean, optional
+        Current default is ``'warn'``, which behaves as ``True`` in addition
+        to raising a warning. That default will be changed to ``False``
+        in 0.22.
+        If ``False``, the ``cv_results_`` attribute will not include training
         scores.
```

Comment:
I would also add a note there such as:

Computing training scores is used to get insights on how different parameters settings impact the overfitting/underfitting trade-off. However computing the scores on the training set can be computationally expensive and is not strictly required to select the parameters that yield the best generalization performance.